build.gradle.kts

```diff
@@ -24,13 +25,25 @@ group = "org.jmailen.gradle"
 description = projectDescription
 
 object Versions {
-    const val androidTools = "4.2.2"
-    const val jetbrainsAnnotations = "22.0.0"
+    const val androidTools = "7.0.4"
     const val junit = "4.13.2"
-    const val ktlint = "0.43.0"
+    const val ktlint = "0.43.2"
     const val mockitoKotlin = "4.0.0"
 }
 
+configurations {
+    register("testRuntimeDependencies") {
+        extendsFrom(compileOnly.get())
+    }
+    configureEach {
+        resolutionStrategy.eachDependency {
+            if (requested.group == "org.jetbrains.kotlin" && requested.name.startsWith("kotlin")) {
+                useVersion(getKotlinPluginVersion())
+            }
+        }
+    }
+}
```

Comment:
Ah, forgot to mention that one. That's basically the same as in #201, with the same reasoning, but this time _something_ in tests config uses incompatible version of kotlin, so instead of digging which dependency needs to be bumped I'm propsoing to simply keep them consistent 🤷‍♂️